src/api/java/com/minecolonies/api/entity/CustomGoalSelector.java

```diff
@@ -77,6 +72,7 @@ public CustomGoalSelector(@NotNull final GoalSelector old)
         super(old.profiler);
         importFrom(old);
         super.goals = this.goals;
+        super.profiler = this.profiler;
```

Comment:
do we need the this.profiler at all?

Reply:
ye its used to set the minecraft debug info, and assigned to super incase anything tries to access it on the base minecraft type.